src/Microsoft.TestPlatform.Common/DataCollection/Interfaces/IDataCollectionManager.cs

```diff
@@ -42,7 +42,7 @@ internal interface IDataCollectionManager : IDisposable
         /// <summary>
         /// Raises SessionStart event to all data collectors configured for run.
         /// </summary>
-        /// <returns>Are test case level events required.</returns>
+        /// <returns>bolean value specifying whether test case events are subscribed by datacollectors. Based on this execution process will decide whether to send TestCaseStart and TestCaseEnd events to dataCollector process.</returns>
```

Comment:
* Out of proc data collector: can we reliably find if the testcase start or end events are actually subscribed?
* Inproc data collector: is the cost of sending an event high?